bool request

---

bool success
controller_msgs/FootstepDataListMessage planned_footsteps
